Backpack/src/main/java/net/skyscanner/backpack/starrating/internal/BpkStarRatingBase.kt

```diff
@@ -92,6 +101,9 @@ open class BpkStarRatingBase internal constructor(
     update()
   }
 
+  protected fun getAccessibilityText(): String? =
```

Comment:
this allows a `x out of y` announcement rather than the default x% out of 100% announcement - this only works for 30+, however when using the control and increasing/decreasing stars it does announce this, rather than the percentage